db/schema.rb

```diff
@@ -505,6 +505,7 @@
     t.string "target_type"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.index ["target_id", "target_type"], name: "settings_target"
```

Comment:
Should there be a corresponding migration file for this?

Reply:
yes 💯

Reply:
just added one